src/core/Akka.Cluster/Cluster.cs

```diff
@@ -140,27 +143,26 @@ public Cluster(ActorSystemImpl system)
             _readView = new ClusterReadView(this);
 
             // force the underlying system to start
-            _clusterCore = GetClusterCoreRef().Result;
-
-            system.RegisterOnTermination(Shutdown);
+            _ = Task.Run(async () =>
+            {
+                try
+                {
+                    _clusterCore = await _clusterDaemons.Ask<IActorRef>(new InternalClusterAction.GetClusterCoreRef(this), System.Settings.CreationTimeout).ConfigureAwait(false);
+                    clusterCoreTaskSource.SetResult(_clusterCore);
 
-            LogInfo("Started up successfully");
-        }
+                    system.RegisterOnTermination(Shutdown);
+                    _log.Info("Started up successfully");
```

Comment:
Needs to be a `LogInfo` call here - passes in context about the current node member address which is helpful for debugging purposes.